README.md

```diff
@@ -79,7 +79,7 @@ To use Thunder on your local machine:
 
 ```bash
 # install nvFuser which installs the matching nightly PyTorch
-pip install --pre 'nvfuser-cu121[torch]' --extra-index-url https://pypi.nvidia.com
+pip install --pre nvfuser-cu121-torch24
```

Comment:
`nvfuser-...-torch24` builds against stable release of torch 2.4. I think we also need to update the line above "which installs the matching ~~nightly~~ stable"

Reply:
Yes, thank you!